sdk/core/core-http/src/policies/tracingPolicy.ts

```diff
@@ -74,10 +74,12 @@ export class TracingPolicy extends BaseRequestPolicy {
     try {
       const path = URLBuilder.parse(request.url).getPath() || "/";
 
+      // Passing spanOptions as part of tracingOptions to maintain compatibility with the previous version of core-tracing.
+      // We can pass this as a separate parameter once we upgrade to the latest core-tracing.
```

Comment:
Should we add more information related to which versions this might be relevant to?